CMakeLists.txt

```
@@ -0,0 +1,252 @@
+# SPDX-License-Identifier: Unlicense
+#
+# LibTomMath, a free open source portable number theoretic multiple-precision
+# integer (MPI) library written entirely in C.
+#
+
+cmake_minimum_required(VERSION 3.10)
+
+project(tommath
+    VERSION 1.2.0
+    DESCRIPTION "A free open source portable number theoretic multiple-precision integer (MPI) library written entirely in C."
+    HOMEPAGE_URL "https://www.libtom.net/LibTomMath"
+    LANGUAGES C)
+
+# package release version
+# bump if re-releasing the same VERSION + patches
+# set to 1 if releasing a new VERSION
+set(PACKAGE_RELEASE_VERSION 1)
+
+#-----------------------------------------------------------------------------
+# Include cmake modules
+#-----------------------------------------------------------------------------
+include(GNUInstallDirs)
+include(CheckIPOSupported)
+include(CMakePackageConfigHelpers)
+# default is "No tests"
+option(BUILD_TESTING "" OFF)
+include(CTest)
```

If I read the [`CTest`](https://cmake.org/cmake/help/latest/module/CTest.html) documentation correctly, the `BUILD_TESTING` option is created by the module.
```suggestion
include(CTest)
```

---

Yes, and as stated in the comment above that `option()` statement we want it to be turned `Off` by default, while CTest declares it as `On`

---

Oops! Sorry! I didn't read that line.